packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -165,27 +151,33 @@ function useProcessMedia({
           // Ignore for now.
           return;
         }
+
+        const additionalData = {
+          original_id: oldResource.id,
+          web_stories_is_muted: oldResource.isMuted,
+        };
+
+        if (oldResource?.baseColor) {
+          additionalData.meta.web_stories_base_color = oldResource.baseColor;
```

Comment:
Pretty sure this will throw an error because `meta` is not defined.

You can try this yourself by running this in the console:

```js
const foo = { bar: 123 }; 
foo.baz.barbar = 456; // Uncaught TypeError: Cannot set properties of undefined (setting 'barbar')
```

Reply:
Migration of base color now happens php / rest api.

Reply:
@spacedmonkey Curious, what's the reason for this change?

IIRC you previously mentioned running it client-side is more safe because the base color could change after trimming for example.

Reply:
When we added server side migration, it was to stop stuff like this. Migrating on the server ends up with cleaner logic in javascript and no repeated code.